lib/developer_portal/lib/liquid/tags/disable_client_cache.rb

```diff
@@ -0,0 +1,21 @@
+module Liquid
+  module Tags
+
+    class DisableClientCache < Liquid::Tags::Base
+      example "Disable browser cache for this screen", %{
+        <html>
+          <head>
+            {% disable_client_cache %}
+          </head>
+        </html>
+      }
+
+      desc "Adds HTTP headers to disable the browser cache for the current screen."
+      def render(context)
+        controller = context.registers[:controller].dup
+        controller.disable_client_cache
```

Comment:
why dup instead of `context.registers[:controller].disable_client_cache`?

Reply:
@jlledom I think this one is for you ⬆️

Reply:
Because I copied it from: https://github.com/3scale/porta/blob/25fa9bc53bdd3eeb8f2a9c8e882f771c85c3c3cb/lib/developer_portal/lib/liquid/tags/csrf.rb#L15

But you're right, `.dup` is not needed here.